app/tests/src/index.js

```diff
@@ -1,28 +1,31 @@
 describe('AdaLite Test Suite', () => {
   require('./common/setup-test-config')
 
-  describe('CBOR', () => {
-    require('./cbor')
-  })
-  describe('Mnemonic Crypto Provider', () => {
-    require('./cardano-wallet-secret-crypto-provider')
-  })
-  describe('Address Manager', () => {
-    require('./address-manager')
-  })
-  describe('Blockchain Explorer', () => {
-    require('./blockchain-explorer')
-  })
-  describe('Cardano Wallet', () => {
-    require('./cardano-wallet')
-  })
-  describe('Import/Export Wallet as JSON', () => {
-    require('./keypass-json')
-  })
-  describe('Actions', () => {
-    require('./actions/actions')
-  })
-  describe('Shelley testnet', () => {
-    require('./shelley')
+  // describe('CBOR', () => {
+  //   require('./cbor')
+  // })
+  // describe('Mnemonic Crypto Provider', () => {
+  //   require('./cardano-wallet-secret-crypto-provider')
+  // })
+  // describe('Address Manager', () => {
+  //   require('./address-manager')
+  // })
+  // describe('Blockchain Explorer', () => {
+  //   require('./blockchain-explorer')
+  // })
+  // describe('Cardano Wallet', () => {
+  //   require('./cardano-wallet')
+  // })
+  // describe('Import/Export Wallet as JSON', () => {
+  //   require('./keypass-json')
+  // })
+  // describe('Actions', () => {
+  //   require('./actions/actions')
+  // })
+  // describe('Shelley testnet', () => {
+  //   require('./shelley')
+  // })
```

Comment:
basically, I would try to keep the tests that make sense and no ned to comment the rest, just delete them and we can always retrieve them from history rather than having dead code